.travis.yml

```diff
@@ -67,19 +74,19 @@ install:
     fi
   - if [ "$TRAVIS_OS_NAME" = "osx" ]; then
       misc/travis/scons-local-osx.sh;
-      misc/travis/ccache-osx.sh;
+      `#misc/travis/ccache-osx.sh`;
     fi
   - if [ "$TRAVIS_OS_NAME" = "osx" ] && [ "$GODOT_TARGET" = "android" ]; then
       misc/travis/android-tools-osx.sh;
     fi
 
 before_script:
   - if [ "$TRAVIS_OS_NAME" = "linux" ]; then
-      export CCACHE="/usr/bin/ccache";
+      `#export CCACHE="/usr/bin/ccache"`;
```

Comment:
What does that change do? In my local shell it doesn't work.

Reply:
Ah it's just a hack to comment it out... Just remove stuff instead of commenting it.

Reply:
@akien-mga I removed the ccache stuff from .travis.yml as you pointed out. Thanks.